applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/eigensolver_strategy.hpp

```diff
@@ -562,8 +562,8 @@ class EigensolverStrategy
                     empty = false;
                     break;
                 }
-            if(empty == true)
```

Comment:
@KratosMultiphysics/dem i would suggest you to comment out the whole loop on lines 553-567